pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "nOAuth"
+version = "0.1.0"
+description = "A simple oauth server that does NO real authentication and helps you test systems that depend on OAuth"
+authors = ["Ben Adida <ben@voting.works>"]
```

Comment:
I've been using `VotingWorks Eng <eng@voting.works>`, fwiw.

Reply:
ooooo didn't know about that, thanks

Reply:
oh yeah, good call :)